app/dashboard/src/modules/payments/components/PlanSelector/components/PlanSelectorDialog.tsx

```diff
@@ -145,7 +155,7 @@ export function PlanSelectorDialog(props: PlanSelectorDialogProps) {
                   form={form}
                   name="period"
                   // eslint-disable-next-line @typescript-eslint/no-magic-numbers
-                  items={[12, 36]}
+                  items={[1, 12, 36]}
```

Comment:
monthly is back?

Reply:
No! I guess it went back after a rebase. Thanks for noticing!